docs/sources/next/shared/statsd-output.md

```diff
@@ -0,0 +1,11 @@
+---
+title: StatsD module admonition
+---
+
+{{% admonition type="warning" %}}
+
+The built-in StatsD output has been deprecated on k6 v0.47.0 and [scheduled to remove in v0.55.0](https://github.com/grafana/k6/pull/3849). You can continue to use this feature by using the [xk6-output-statsd extension](https://github.com/LeonAdato/xk6-output-statsd), and this guide has been updated to include instructions for how to use it.
```

Comment:
Maybe also mention the OpenTelemetry output, WDYT?

Reply:
Probably good idea, but also probably a good idea. Not certain I will do it in the admonition as it goes for all of the specific products (amazon cloudwatch, datadog ...) 

But it might actually be better if they are edited to recommend using somethign that *is not* statsd. Like for example the opentelemetry output